pkg/gomod/file/testdata/go.0.mod

```diff
@@ -0,0 +1,5 @@
+module "my/thing"
```

Comment:
maybe one more with different order, parser should look it up

Reply:
Fixed in https://github.com/gomods/athens/pull/72/files#diff-ae9bcd418668c90e559d1c8331670d4c